lib/hammer_cli_foreman_puppet.rb

```diff
@@ -4,12 +4,14 @@ module HammerCLIForemanPuppet
 
   require 'hammer_cli_foreman_puppet/version'
   require 'hammer_cli_foreman_puppet/puppet_references'
+  require 'hammer_cli_foreman_puppet/commands'
   require 'hammer_cli_foreman_puppet/command_extensions'
   require 'hammer_cli_foreman_puppet/option_sources'
   require 'hammer_cli_foreman_puppet/associating_commands'
 
   # Puppet commands
   require 'hammer_cli_foreman_puppet/smart_class_parameter'
+  require 'hammer_cli_foreman_puppet/id_resolver'
```

Comment:
This needs to go up (above commands) such that it is available in all commands, otherwise you need to include it in `hammer_cli_foreman_puppet/commands` and everywhere it is needed. 

Is there a cleaner way to resolve dependencies?